src/gui/advancedsettings.cpp

```diff
@@ -61,6 +61,7 @@ namespace
     {
         // qBittorrent section
         QBITTORRENT_HEADER,
+        RESUME_DATA_STORAGE,
```

Comment:
What about also exposing this on the WebUI? Or would you rather leave that for another PR? Typically we prefer exposing on both interfaces from the start, right?

Reply:
>What about also exposing this on the WebUI?

I not intend to do it at experimental stage of this feature. I don't want to complicate the feature adaptation process.
>Typically we prefer exposing on both interfaces from the start, right?

As long as we do not have a permanent web UI maintainer, I would not impose such burdensome responsibilities on us. Only in the case of breaking changes.